### PR TITLE
fix: Address blank screen issue

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue #767 and #813.

## Relevant technical choices

As mentioned in https://github.com/google/site-kit-wp/issues/767#issuecomment-553586784, I think the issue here is the `pageTitle` variable being improperly set, based on the tracebacks/error outputs reported by users. I'm waiting for affected sites to try this patch, or to get a build that errors in the same way users are reporting to see if this is the error they've been seeing…

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3868,6 +3868,7 @@
         "spawn-promise": "^0.1.8",
         "style-loader": "^0.23.1",
         "svg-url-loader": "^2.3.2",
+        "terser-webpack-plugin": "^1.2.1",
```
Comment (felixarntz) [inline package-lock.json]: All those changes in `package-lock.json` seem unrelated. Could you clarify why they're here, or remove them otherwise? :)

File: assets/js/modules/search-console/dashboard/dashboard-widget-sitestats.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ class SearchConsoleDashboardWidgetSiteStats extends Component {
 	setOptions() {
 		const { selectedStats, series, vAxes } = this.props;
 
-		const pageTitle = '' === googlesitekit.pageTitle ? '' : sprintf( __( 'Search Traffic Summary for %s', 'google-site-kit' ), decodeHtmlEntity( googlesitekit.pageTitle ) );
+		const pageTitle = googlesitekit.pageTitle && googlesitekit.pageTitle.length ? sprintf( __( 'Search Traffic Summary for %s', 'google-site-kit' ), decodeHtmlEntity( googlesitekit.pageTitle ) ) : '';
```
Comment (felixarntz) [inline assets/js/modules/search-console/dashboard/dashboard-widget-sitestats.js]: This should still have a value when there is no page title, e.g. just `__( 'Search Traffic Summary', 'google-site-kit' )`.

Comment (felixarntz) [inline assets/js/modules/search-console/dashboard/dashboard-widget-sitestats.js]: This should still have a value when there is no page title, e.g. just `__( 'Search Traffic Summary', 'google-site-kit' )`.

Comment (felixarntz) [inline assets/js/modules/search-console/dashboard/dashboard-widget-sitestats.js]: This should still have a value when there is no page title, e.g. just `__( 'Search Traffic Summary', 'google-site-kit' )`.

File: package-lock.json (hunk 1 above)
Comment (tofumatt) [reply package-lock.json]: Hmm, nothing intentional, just the result of running `npm install`. I'll try to merge in `develop` and see if it persists…

File: assets/js/modules/search-console/dashboard/dashboard-widget-sitestats.js (hunk 2 above)
Comment (tofumatt) [reply assets/js/modules/search-console/dashboard/dashboard-widget-sitestats.js]: In the past it didn't, but fair enough!
